src/tests/multihost/alltests/test_proxy_provider_krb_auth.py

```diff
@@ -0,0 +1,122 @@
+""" Automation of proxy provider suite
+
+:requirement: IDM-SSSD-REQ : Proxy Provider
+:casecomponent: sssd
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+"""
+from __future__ import print_function
+import pytest
+import paramiko
+import subprocess
+from sssd.testlib.common.utils import sssdTools, SSHClient
+
+
+def execute_cmd(multihost, command):
+    """ Execute command on client """
+    cmd = multihost.client[0].run_command(command)
+    return cmd
+
+
+@pytest.mark.usefixtures('setup_sssd_krb',
+                         'create_posix_usersgroups',
+                         'sssdproxyldap_test')
+@pytest.mark.tier1_3
+class TestProxy(object):
+    """
+    This is test case class for ldap proxy suite
+    """
+    def test_proxy_lookup(self, multihost, backupsssdconf):
+        """
+        :title: Proxy lookup and kerberos auth
+        :id: 5c4c55b8-0cac-47d0-aa23-d057b790e18e
+        :steps:
+          1. Check ldap user has access to client machine
+          2. Proxy lookup and kerberos auth
+          3. Change expired password
```

Comment:
It looks like the `Change expired password` test is moved into a dedicated test.

Reply:
Fixed